pkg/commands/oscommands/os.go

```diff
@@ -267,6 +267,13 @@ func (c *OSCommand) CopyToClipboard(str string) error {
 	escaped := strings.Replace(str, "\n", "\\n", -1)
 	truncated := utils.TruncateWithEllipsis(escaped, 40)
 	c.LogCommand(fmt.Sprintf("Copying '%s' to clipboard", truncated), false)
+		if c.UserConfig.OS.CopyToClipboardCmd != "" {
+				cmdStr := utils.ResolvePlaceholderString(c.UserConfig.OS.CopyToClipboardCmd, map[string]string{
+					"text": c.Cmd.Quote(str),
```

Comment:
I see that text is quoted here but that we're also quoting it in the example command above. Does that still work?

Reply:
Good catch. The example worked fine because there are another pair of outer enclosing quotes, but was confusing. Simplified.